lib/morph/database.rb

```diff
@@ -128,7 +128,7 @@ def sql_query_safe(query, readonly = true)
 
     # Returns 0 if table doesn't exists (or there is some other problem)
     def no_rows(table = table_names.first)
-      q = sql_query_safe("select count(*) from '#{table}'")
+      q = sql_query_safe("select count(*) from \"#{table}\"")
```

Comment:
Not enough rubyfu here to apply same to the other affected queries in this file. Obv I have no issue with it